docker/ceph/centos/Dockerfile

```
@@ -2,9 +2,9 @@ ARG CENTOS_VERSION=8
 FROM centos:$CENTOS_VERSION as ceph-base
 ARG CENTOS_VERSION
 
-# Centos met EOL and the content of the CentOS 8 repos has been moved to vault.centos.org
+# Centos met EOL and the content of the CentOS 8 repos has been moved to vault.epel.cloud
 RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-Linux-*
-RUN sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-Linux-*
+RUN sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.epel.cloud|g' /etc/yum.repos.d/CentOS-Linux-*
```

I'm getting `301 Moved Permanently` on `curl -v http://vault.centos.org` , it's basically getting redirected to https, so can u instead try this?
```suggestion
RUN sed -i 's|#baseurl=http://mirror.centos.org|baseurl=https://vault.centos.org|g' /etc/yum.repos.d/CentOS-Linux-*
```

---

@avanthakkar @nizamial09 this worked!!

---

`RUN sed -i 's|#baseurl=http://vault.centos.org|baseurl=http://vault.epel.cloud|g' /etc/yum.repos.d/CentOS-Linux-*`
this should be the change but I made it wrong. I'll try a bit with https too because for me it was not accessible couple of times.